tests/st/codegen/test_paged_attention.py

```
@@ -74,31 +75,33 @@ class QKMatmulProgram:
-                self, qi: pl.Tensor[[16, 16], pl.FP32], kj_t: pl.Tensor[[16, 16], pl.FP32]
-            ) -> pl.Tensor[[16, 16], pl.FP32]:
-                out_sij: pl.Tensor[[16, 16], pl.FP32] = pl.create_tensor([16, 16], dtype=pl.FP32)
+                self, qi: pl.Tensor[[16, 128], pl.BF16], kj_t: pl.Tensor[[128, 128], pl.BF16]
+            ) -> pl.Tensor[[128, 128], pl.FP32]:
+                out_sij: pl.Tensor[[128, 128], pl.FP32] = pl.create_tensor([128, 128], dtype=pl.FP32)
                 out_sij = self.qk_matmul(qi, kj_t, out_sij)
                 return out_sij
 
         return QKMatmulProgram
 
     def compute_expected(self, tensors, params=None):
```

Ruff (0.15.2)

[warning] Unused method argument: `params`

(ARG002)